tests/ui_tests/test_header_and_footer.py

```diff
@@ -0,0 +1,112 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=W0621
+
+import allure
+
+from tests.ui_tests.app.page.admin_intro.admin_intro_page import AdminIntroPage
+from tests.ui_tests.app.page.admin_setttings_page.admin_settings import AdminSettingsPage
+from tests.ui_tests.app.page.bundle_list_page.bundle_list import BundleListPage
+from tests.ui_tests.app.page.cluster_list_page.cluster_list import ClusterListPage
+from tests.ui_tests.app.page.common.base_page import (
+    BasePageObject,
+    PageHeader,
+    PageFooter,
+)
+from tests.ui_tests.app.page.host_list_page.host_list import HostListPage
+from tests.ui_tests.app.page.hostprovider_list_page.hostprovider_list import ProviderListPage
+from tests.ui_tests.app.page.job_list_page.job_list import JobListPage
+from tests.ui_tests.app.page.login.login_page import LoginPage
+from tests.ui_tests.app.page.profile_page.profile import ProfilePage
+from tests.ui_tests.utils import (
+    wait_for_new_window,
+    close_current_tab,
+)
+
+
+def test_check_header_tabs_for_authorised_user(app_fs, auth_to_adcm):
+    header = PageHeader(app_fs.driver, app_fs.adcm.url)
+
+    header.click_arenadata_logo_in_header()
+    intro_page = AdminIntroPage(app_fs.driver, app_fs.adcm.url)
+    header.wait_url_contains_path(intro_page.path)
+
+    header.click_cluster_tab_in_header()
+    cluster_page = ClusterListPage(app_fs.driver, app_fs.adcm.url)
+    header.wait_url_contains_path(cluster_page.path)
+
+    header.click_hostproviders_tab_in_header()
+    provider_page = ProviderListPage(app_fs.driver, app_fs.adcm.url)
+    header.wait_url_contains_path(provider_page.path)
+
+    header.click_hosts_tab_in_header()
+    host_page = HostListPage(app_fs.driver, app_fs.adcm.url)
+    header.wait_url_contains_path(host_page.path)
+
+    header.click_jobs_tab_in_header()
+    job_page = JobListPage(app_fs.driver, app_fs.adcm.url)
+    header.wait_url_contains_path(job_page.path)
+
+    header.click_bundles_tab_in_header()
+    bundle_page = BundleListPage(app_fs.driver, app_fs.adcm.url)
+    header.wait_url_contains_path(bundle_page.path)
+
+    header.click_job_block_in_header()
+    job_page = JobListPage(app_fs.driver, app_fs.adcm.url)
+    header.wait_url_contains_path(job_page.path)
+    header.check_job_popup()
+
+
+def test_check_header_help_links_for_authorised_user(app_fs, auth_to_adcm):
+    params = {
+        "help_link": "t.me/joinchat/",
+        "docs_link": "docs.arenadata.io/adcm/"
+    }
```

Comment:
Move outside test function.

Reply:
why? now this is just for this func

Reply:
To keep test data separate of test itself.

Reply:
again why?

Reply:
Prepare data using parametrization and or const  -> execute steps in test.
Also it looks like const vars.

Reply:
it is this function consts, using parametrization will be a complication and worsed read